include/records/RecYAMLDefs.h

```diff
@@ -73,7 +73,7 @@ struct CfgNode {
 
   /// @brief Append field name in order to build up the record name.
   void
-  append_field_name() const
```

Comment:
Can the `mutable` qualifier on `CfgNode::_legacy` be removed now?

Reply:
Yes!! pretty much the whole point 🤣  , good catch Josiah!! appreciated mate.